rllib/algorithms/dreamerv3/dreamerv3.py

```diff
@@ -28,24 +26,29 @@
 from ray.rllib.core import DEFAULT_MODULE_ID
 from ray.rllib.core.columns import Columns
 from ray.rllib.core.rl_module.rl_module import SingleAgentRLModuleSpec
+from ray.rllib.execution.rollout_ops import synchronous_parallel_sample
```

Comment:
This is the most important file to review, the rest is mere "translation" from the existing tf version to torch.